src/Workspaces/Core/Portable/Execution/AbstractReferenceSerializationService.cs

```diff
@@ -101,13 +101,17 @@ public Checksum CreateChecksum(AnalyzerReference reference, CancellationToken ca
                         WriteUnresolvedAnalyzerReferenceTo(unresolved, writer);
                         break;
 
+                    case AnalyzerReference analyzerReference when analyzerReference.GetType().FullName == "Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.VisualStudioUnresolvedAnalyzerReference":
```

Comment:
@heejaechang it was (very recently) added in https://github.com/dotnet/roslyn/pull/20655.  May still only be in the `dev15.3.x` branch